pipeline/load/belonging.sql

```diff
@@ -0,0 +1,11 @@
+DROP TABLE IF EXISTS belonging;
+CREATE TABLE belonging (
+    area_number INTEGER,
+    segment TEXT,
+    value REAL,
+    std_error REAL,
+    period_start_year INTEGER,
+    period_end_year INTEGER,
+    period TEXT,
+    PRIMARY KEY(area_number)
```

Comment:
This works! However, the primary key should be the column(s) that uniquely identify rows in the table.

In this case, I would suggest:

```
PRIMARY KEY (area_number, period_end_year)
```

I left `segment` out of the primary key because right now we only have one segment: the entire population.